requirements/base.txt

```diff
@@ -5,6 +5,7 @@ cffi==1.12.3
 cryptography==2.7
 Django==2.2.5
 django-cors-headers==3.1.0
+django_countries
```

Comment:
nit: add version with this package.

Ideally we should provide the range of package version but for now its fine to pin it to some specific version.

Reply:
Done.